core/templates/services/platform-feature.service.ts

```diff
@@ -162,6 +162,7 @@ export class PlatformFeatureService {
         await this.loadFeatureFlagsFromServer());
 
       this.saveResults();
+    // Unknown type is used because we don't know which type pf error thrown.
```

Comment:
```suggestion
    // Unknown type is used because we don't know which type of error is thrown.
```

Ditto everywhere else in the PR where applicable (this comment comes out exactly to 80 characters, so it should fit where needed).

Reply:
Done